examples/experimental/ProductionMonitoring/stream_table_api.ipynb

```diff
@@ -0,0 +1,170 @@
+    "\n",
+    "```python\n",
+    "st.log({\"one_column_name\" : \"value_a\", \"another_column_name\" : 7})\n",
+    "st.log([{\"one_column_name\" : \"value_b\", \"another_column_name\" : 19},\n",
+    "        {\"one_column_name\" : \"value_c\", \"another_column_name\" : 28},\n",
+    "        {\"one_column_name\" : \"value_d\", \"another_column_name\" : 36}]\n",
+    "```\n",
+    "\n",
+    "The first call to `.log()` will return a Weave Panel URL, where you can view, edit, and save the resulting StreamTable as a Weave Board, of the form:\n",
+    "\n",
+    "View data at: https://weave.wandb.ai/?exp=get%28%0A++++%22wandb-artifact%3A%2F%2F%2Fstacey%2Fmesa%2Fmy_stream_table%3Alatest%2Fobj%22%29%0A++.rows\n",
+    "\n",
+    "Subsequent log calls will silently append these rows to the StreamTable instance.\n",
+    "\n",
+    "In a notebook, the StreamTable variable on a line by itself will return a Weave Panel view of the StreamTable. The StreamTable will contain all the logged columns and their values, as well as a `timestamp` column indicating when the row was logged. By default, rows will be ordered by oldest first. You can modify a StreamTable Panel as you would a wandb.Table from the UI: sort, filter, group, etc.\n",
+    "\n",
+    "**Note: Column display order is currently non-deterministic** over repeated views of the Panel. If you would like to modify the order, open the StreamTable Panel in a new window as a Board and edit/save a Board from this seed panel. There are two options to achieve this:\n",
+    "* via the weave.wandb.ai/?exp=... URL\n",
+    "* via \"Open in new tab\" arrow button, revealed in the menu when you hover on the right side of a StreamTable panel displayed in the notebok)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b928c2e0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# log data to the StreamTable as a dictionary or list of dictionaries\n",
+    "st.log({\"col_a\" : \"1\", \"col_b\" : \"17\", \"col_c\" : \"42\"})\n",
+    "\n",
+    "# show the StreamTable\n",
+    "st"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "83cec916",
+   "metadata": {},
+   "source": [
+    "## Step 3: Log more data & explore the results!\n",
+    "\n",
+    "Continue logging as much data as you like. If you save the StreamTable Panel as a Board, the Board will continue to update as you send more data to the same StreamTable instance.\n",
+    "\n",
+    "### Important usage notes\n",
+    "\n",
+    "Some details to keep in mind for this early iteration of the StreamTables API:\n",
+    "\n",
+    "* **columns must match exactly in each log**: each dictionary passed to `.log()` must contain a key for every column in the StreamTable instance. Skipping a key, even if its value is null, or adding a new key/column name after the first log call, may break the StreamTable. \n",
```

Comment:
is this true? I thought we dynamically handle the column types expanding. I would not write this, it feels like a bug we should fix asap if true.

Reply:
this is true, this is me documenting gotchas.